### PR TITLE
Added Folding!

### PR DESCRIPTION
**Pull request by Blazin3330**
You can now use Sk-VSC to categorise code inside your skripts.

Example:
```
#!FOLD [NAME]
  > YOUR CODE HERE
#!UNFOLD
```

skvsc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,13 +33,23 @@ Adding your own theme to this syntax (Sublime Text 3 version) is pretty easy
 
 
 
+# Chanelog (v2.5.4)
```
Comment (AyhamAl-Ali) [inline README.md:36]:
```suggestion
# Chanelog (v2.6.4)
```

File: README.md [hunk 2]
```diff
@@ -33,13 +33,23 @@ Adding your own theme to this syntax (Sublime Text 3 version) is pretty easy
...
+
+> Added folding! Useful for categorising your code!
+
+Example:
+```
+#!FOLD [NAME]
+   > YOUR CODE HERE
```
Comment (AyhamAl-Ali) [inline README.md:43]:
```suggestion
> YOUR CODE HERE
```

File: VSC/src/CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,14 @@
+# Changelog (v2.5.4)
```
Comment (AyhamAl-Ali) [inline VSC/src/CHANGELOG.md:1]:
```suggestion
# Changelog (v2.6.4)
```

File: VSC/src/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,14 @@
...
+
+> Added folding! Useful for categorising your code!
+
+Example:
+```
+#!FOLD [NAME]
+   > YOUR CODE HERE
```
Comment (AyhamAl-Ali) [inline VSC/src/CHANGELOG.md:8]:
```suggestion
> YOUR CODE HERE
```

File: VSC/src/language-configuration.json [hunk 5]
```diff
@@ -1,30 +1,31 @@
 {
-    "comments": {
-        // symbol used for single line comment. Remove this entry if your language does not support line comments
-        "lineComment": "#",
-        // symbols used for start and end a block comment. Remove this entry if your language does not support block comments
-        // "blockComment": []
-    },
-    // symbols used as brackets
```
Comment (AyhamAl-Ali) [inline VSC/src/language-configuration.json:8]: Keep this comment :\

File: VSC/src/language-configuration.json [hunk 6]
```diff
@@ -1,30 +1,31 @@
...
-    "brackets": [
-        ["{", "}"],
-        ["[", "]"],
-        ["(", ")"]
-    ],
-    // symbols that are auto closed when typing
```
Comment (AyhamAl-Ali) [inline VSC/src/language-configuration.json:14]: Keep this comment :\

File: VSC/src/language-configuration.json [hunk 7]
```diff
@@ -1,30 +1,31 @@
...
-    "autoClosingPairs": [
-        ["{", "}"],
-        ["[", "]"],
-        ["(", ")"],
-        ["\"", "\""],
-        ["'", "'"]
-    ],
-    // symbols that can be used to surround a selection
```
Comment (AyhamAl-Ali) [inline VSC/src/language-configuration.json:22]: Keep this comment :\

**Review by AyhamAl-Ali — CHANGES_REQUESTED** [review]
Good Job! 🚀

Just few minor things

**Review by Blazin3330 — COMMENTED** [review]
Updated

File: VSC/src/language-configuration.json (hunk 5 above)
Comment (AyhamAl-Ali) [reply VSC/src/language-configuration.json:8]: Unsolved request.

File: VSC/src/language-configuration.json (hunk 6 above)
Comment (AyhamAl-Ali) [reply VSC/src/language-configuration.json:14]: Unsolved request.

**Review by AyhamAl-Ali — CHANGES_REQUESTED** [review]

**Review by AyhamAl-Ali — APPROVED** [review]

**Pull request merged**